### DISCUSSION TITLE
2024 DELPH-IN poll (please fill in by Jan 5th)

### DISCUSSION THREAD

**Discussion by fcbond** [delph-in/docs#143] [Announcements]
G'day,

we will host the summit in Olomouc, back-to-back with HPSG.

Please let us know your availability in this spreadsheet, with 1 for can come; 0 for can't come and 0.5 for may be able to come.

https://docs.google.com/spreadsheets/d/1KtHyiU3yp4v4A44TUZOb6K6M2rdVsxzLfzIl4HHDV9A/edit?usp=sharing

The dates are July to the first week of August.  Currently, I would prefer earlier (so as to not clash with ESSLI and ACL) but let's see what works for everyone.

Looking forward to seeing you in Olomouc,

Francis and Joanna

**Comment by olzama** [comment]
tagging all @delph-in/participants

**Comment by arademaker** [comment]
what is the `Ta Le` in cell A5?

**Comment by fcbond** [comment]
On Tue, 9 Jan 2024 at 05:21, Alexandre Rademaker ***@***.***>
wrote:

> what is the Ta Le in cell A5?
>
It  should have been (and now is) July 21-26



> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/delph-in/discussions/143#discussioncomment-8054495>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAIPZRRZADOYO37E5TETN4TYNRBMJAVCNFSM6AAAAABBE5SVMOVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DANJUGQ4TK>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Francis Bond <https://fcbond.github.io/>

**Comment by curtosis** [comment]
I apparently don't have the right creds to add a column; something to do with protected elements? I can add rows just fine....
(Yes, I'm behind schedule -- been tied up moving house.) 
Solution: added a column after the sum and expanded the sum formula.
